lib/services/ios-debug-service.ts

```diff
@@ -50,21 +50,8 @@ export class IOSDebugService extends DebugServiceBase implements IPlatformDebugS
 			debugOptions.emulator = true;
 		}
 
-		if (!debugOptions.justlaunch) {
-			let projectName = debugData.projectName;
-			if (!projectName && debugData.projectDir) {
-				const projectData = this.$projectDataService.getProjectData(debugData.projectDir);
-				projectName = projectData.projectName;
-			}
-
-			if (projectName) {
-				this.$deviceLogProvider.setProjectNameForDevice(debugData.deviceIdentifier, projectName);
-			}
-
-			await this.device.openDeviceLogStream({ predicate: IOS_LOG_PREDICATE });
-		}
-
 		await this.$iOSDebuggerPortService.attachToDebuggerPortFoundEvent(this.device, debugData, debugOptions);
+		await this.startDeviceLogProcess(debugData, debugOptions);
```

Comment:
I think this should be executed prior to `await this.$iOSDebuggerPortService.attachToDebuggerPortFoundEven...`, because we depend on the logs to get the port.